base_marker:
  position:
    x: 1.337
    y: 0.795
    z: 0.0
camera_pose:
  position:
    x: 1.5943332826963021
    y: 0.7452710532440687
    z: 2.337244362239359
  orientation:
    x: -0.998811632507675
    y: -0.006623176611360882
    z: 0.024962360733968892
    w: 0.04133203171288797
